agent/php_execute.c

```diff
@@ -2071,13 +1876,7 @@ static void nr_php_instrument_func_begin(NR_EXECUTE_PROTO) {
     nr_php_observer_attempt_call_cufa_handler(NR_EXECUTE_ORIG_ARGS);
   }
   wraprec = nr_php_get_wraprec(execute_data->func);
-  /*
```

Comment:
Why is this removed?

Reply:
Because the metadata is no longer stored in the segment, we need to always start a segment. Previously, when we got to fcall_end, we were able to compare the metadata to the execute_data and see if a segment was started for the given execute_data.

Without the metadata and without always creating a segment, we would have no way of determining whether the active segment was actually for the given fcall_end

Reply:
This needs to be investigated, because without these checks, the sheer amount of segment initializations and deinits it too computationally costly.

Reply:
My original reply still holds true and the performance cost has been addressed in my other PRs. (https://github.com/newrelic/newrelic-php-agent/pull/776 and https://github.com/newrelic/newrelic-php-agent/pull/801)